client/src/pages/events/description-view.tsx

```diff
@@ -7,6 +7,7 @@ export function DescriptionView({ description }: { description: string }) {
   return (
     <div
       className="ql-editor"
+      style={{ padding: "0", fontWeight: "500", fontSize: "18px", color: "#202020" }}
```

Comment:
use styled for styles override. and let's remove side paddings only. the rest look fine already